wire-ios-sync-engine/Source/Use cases/E2EIdentityCertificateUpdateStatusUseCase.swift

```diff
@@ -19,96 +19,108 @@
 import Foundation
 
 public enum E2EIdentityCertificateUpdateStatus {
-    case noAction, reminder, block
+    // Alert was already shown within snooze period, so do not remind user to update certificate
+    case noAction
+
+    // Alert was not  shown within snooze period, so remind user to update certificate
+    case reminder
+
+    // certificate expired so soft block user to update certificate
+    case block
 }
 
 // sourcery: AutoMockable
-public protocol E2EIdentityCertificateUpdateStatusProtocol {
+public protocol E2EIdentityCertificateUpdateStatusUseCaseProtocol {
     func invoke() async throws -> E2EIdentityCertificateUpdateStatus
 }
-public extension Notification.Name {
-    static let checkForE2EICertificateStatus = NSNotification.Name("CheckForE2EICertificateStatus")
-}
 
-final public class E2EIdentityCertificateUpdateStatusUseCase: E2EIdentityCertificateUpdateStatusProtocol {
+final public class E2EIdentityCertificateUpdateStatusUseCase: E2EIdentityCertificateUpdateStatusUseCaseProtocol {
 
-    private let e2eCertificateForCurrentClient: GetE2eIdentityCertificatesUseCaseProtocol
+    private let e2eiCertificateForCurrentClient: GetE2eIdentityCertificatesUseCaseProtocol
```

Comment:
as convention we call the useCases variables the same name but we drop the UseCase suffix

```suggestion
    private let getE2eIdentityCertificates: GetE2eIdentityCertificatesUseCaseProtocol
```